etcdutl/snapshot/util.go

```diff
@@ -20,12 +20,23 @@ import (
 
 type revision struct {
 	main int64
+	sep  byte
 	sub  int64
 }
 
+// The binary representation of the revision is 17 bytes:
```

Comment:
Dob't think that's true. I remember that there might be 18byte to mark a tombstone. I will need to confirm, but we should not copy the code to parsing/dumping revisions lightly.

Reply:
https://github.com/etcd-io/etcd/blob/7444985cab75e3d8ef4ef5ffe71634547556aa68/server/storage/mvcc/kvstore.go#L523-L532